org.eclipse.m2e.pde.target/src/org/eclipse/m2e/pde/target/MavenTargetLocation.java

```diff
@@ -208,9 +208,7 @@ private void generateFeature(TargetBundles bundles, boolean source) throws CoreE
 		} else {
 			bundleFilter = Predicate.not(bundleFilter);
 		}
-		Iterable<TargetBundle> featurePlugins = bundles.bundles() //
-				.filter(e -> !isExcluded(e.getKey()) && !isIgnored(e.getKey()))//
```

Comment:
ignored artifacts are not added to TargetBundles.bundles so we don't have to check if they are ignored.